### PR TITLE
add init-project

### PR DESCRIPTION
**Pull request by 18a6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# mev-mantis
+## Etherscan API consumption in Golang
```
Comment (achab) [inline README.md:1]: good to have changed the name of the current project. could you also add commands to:
- install
- compile
- run

File: extract.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func get_etherscan_key() string {
```
Comment (achab) [inline extract.go:12]: with Go, the naming convention is CamelCase (and not snake_case like for your function). see: https://go.dev/doc/effective_go#mixed-caps

File: extract.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	etherscanKey, exists := os.LookupEnv("ETHERSCAN_KEY")
+
+	if exists {
+		return etherscanKey
+	}
+	return ""
+}
+
+// returns the Ether balance of a given address
+func get_balance(address, tag string) string {
```
Comment (achab) [inline extract.go:27]: same here, it should be named `getBalance`

File: README.md (hunk 1 above)
Comment (18a6) [reply README.md]: I updated the README.md as follows: navigate to the cloned folder, then run: `go run extract.go` 
Not going to work though until i implement `main()`

**Review by achab — APPROVED** [review]
Great! Feel free to merge the PR!

**Pull request merged**